src/blog/models.py

```diff
@@ -90,6 +90,8 @@ class BlogPage(Page):
     )
     intro = models.CharField(max_length=250)
     body = RichTextField(blank=True)
+    video = models.URLField(null=True, blank=True, help_text="Google drive share link",
+)
```

Comment:
I guess this is fine for now, especially if we are sure we only want to upload one video per post. But we can also think about creating custom blocks: https://docs.wagtail.org/en/stable/advanced_topics/customisation/streamfield_blocks.html

Reply:
As of now, I only see myself posting 1 vlog/video per post. So I think it should be fine
But maybe we can do the custom block as a refactor/upgrade later on if we want it